vaadin-icons-flow-parent/vaadin-icons-flow-integration-tests/src/test/java/com/vaadin/flow/component/icon/tests/IconIT.java

```diff
@@ -123,6 +129,12 @@ private void assertIconProperty(String id, String collection,
         assertIconProperty(findElement(By.id(id)), collection, iconName);
     }
 
+    private void assertHasIconSvg(String id) {
+        WebElement icon = findElement(By.id(id));
+        List<WebElement> svgs = findInShadowRoot(icon, By.tagName("svg"));
+        assertTrue("The icon has no svg inside", svgs.size() == 1);
```

Comment:
Regarding the previous comment, I would also suggest changing `assertTrue` to `Assert.assertTrue`.

`Assert.assertTrue` (~560) seems more frequently used across the repo rather than `assertTrue` (~80).

Reply:
Done.